resources/eventing/charts/nats/values.yaml

```diff
@@ -0,0 +1,12 @@
+# Default values for nats.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+global:
+  install:
+    enabled: false
+  namespace: nats
```

Comment:
We don't need this as kyma-installer is also passing the "--create-namespace <namespace-name>" when installing the chart.

Reply:
It is still needed for "nats-operator-binding".
We will use "--create-namespace" for now, see also README.md.

Reply:
You can use

 ```
{{ .Release.Namespace }}
```

to get the namespace from Helm dynamically.

Reply:
True, but I wanted to let it simple now, cause NATS operator has different behavior depending of the namespace.. :
```
  # Change to the name of the namespace where to install NATS Operator.
  # Alternatively, change to "nats-io" to perform a cluster-scoped deployment in supported versions.
```